tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/ConcreteFunction.java

```diff
@@ -154,78 +178,191 @@ public Signature signature() {
   }
 
   /**
-   * Invokes a function.
-   *
-   * <p>Caller is responsible for closing all Tensors.
-   *
-   * @param arguments list of tensors to pass in input to the function,
-   *                  mapped by their signature name
-   * @return output tensors resulting from the execution of the function,
-   *         mapped by their signature name
+   * Get the name of the function.
    */
-  public Map<String, Tensor> call(Map<String, Tensor> arguments)
-      throws IllegalArgumentException {
+  public String getNativeFunctionName() {
```

Comment:
I left that public because it can matter when working with imported or exported graphs, since the native name is what it shows up as in the graph def.  Maybe change it to `attachedFunctionName` or `graphFunctionName`?

Reply:
or just `getFunctionName()` or `getName()`?

Reply:
I didn't want to collide with `signature.methodName`.  Maybe `getDefinitionName()`?

Reply:
Hey @rnett , we did not closed that one. I'd be ok with `getName()` or `getDefinedName()`, wdyt?

Reply:
`getDefinedName` sounds good to me.